models/state.go

```diff
@@ -43,14 +44,21 @@ func (s State) TransitionAllowed(target State) bool {
 		}
 	case StateUploaded:
 		switch target {
-		case StateUploaded, StatePublishing, StateDeleted:
+		case StateUploaded, StateImporting, StateDeleted:
```

Comment:
The `StatePublishing` is no longer required?

Reply:
Also, added completed and failed states (not used yet, but we will need them in the future)